bin/download-tgz

```diff
@@ -17,11 +17,20 @@ const commands = require('../lib/commands');
 
 program.version(packageJson.version);
 
+const parseConcurrency = (value) => {
```

Comment:
move this function into `lib`, this file should only describe the API of the CLI tool